README.md

```diff
@@ -1,2 +1 @@
-# Review-a-former-project-1
-Reviewing a project from Trial 1 of Microverse
+# coding-challenge-2
```

Comment:
Kindly consider the Microverse README [template](https://github.com/microverseinc/readme-template) and try to customize it personally which will lead your repo to look more professional